.travis.yml

```diff
@@ -21,7 +21,6 @@ before_install:
     - openssl aes-256-cbc -K $encrypted_6a1e0d13907f_key -iv $encrypted_6a1e0d13907f_iv -in id_rsa.enc -out travis-deploy-key -d
     - chmod 600 travis-deploy-key # this key should have push access
     - ssh-add travis-deploy-key
-    - npm config set package-lock false
```

Comment:
Weird that it does still show this file. You should delete this line in the Jenkinsfile instead

Reply:
Made the PR a long time ago when the file still existed, maybe that's why